.circleci/config.yml

```diff
@@ -0,0 +1,33 @@
+version: 2.1
+
+orbs:
+  python: circleci/python@0.2.1
+
+jobs:
+  build-and-test:
+    executor: python/default
+    steps:
+      - checkout
+      - python/load-cache
+      - python/install-deps
+      - python/save-cache
+      - run:
+          name: Ensure code is compliant with black
+          command: pip install black && black . --check
+      - run:
+          name: Install Earthshot package
+          command: sudo python setup.py install
+      - run:
+          name: Read GEE service account information
+          command: echo $service_acct_json > "SERVICE_ACCT.json"
+      - run:
+          name: Test connection to GEE API
+          command: pytest earthshot/tests/test_ee_api.py
+      - run:
+          name: Run full test suite
+          command: pytest . --ignore=earthshot/tests/test_ee_api.py
```

Comment:
Suggest to me that the ignored test might be better to live outside the scope of pytest.

Reply:
The reason it gets ignored there is just that it was already run in the previous step (with the logic that we should fail early if the API connector isn't working); when running locally I opt to run everything together, instead.

Reply:
But dosent the default fail?

Reply:
I think I see the conundrum, that a unique, analogous config is needed local testing for each user. It might be best to assume a default location for the local user to install that... probably outside the repo. Anyway, you probably thought it over. I have not, but my bias is that I tend to rely more on local testing than on CI. So I like to have local testing be as easy as possible and would put the exceptions into this file.

Reply:
You convinced me! I decided to add an `--interactive` option, which creates an interactive GoogleAuth flow if users invoke it. If I was more familiar with adding custom options for pytest, I would alias it so that calling `--interactive` also invokes the `-s` option (which sets `--capture=no`), but I am not sure about a good way to do this.